packages/nimble-components/src/table/specs/README.md

```diff
@@ -1,78 +1,276 @@
-# Nimble Table High Level Design
+# Nimble Table
 
-## Problem Statement
+## Overview
 
-Tables are used widely throughout NI, and are a common component in design systems. Third-party tables currently in use within NI web products have been unsatisfactory on multiple fronts, and so we have found it necessary to introduce a table into the `nimble-components` library.
+The `nimble-table` is a component that offers a way to render tabular data in a variety of ways in each column.
 
-## Links To Relevant Work Items and Reference Material
+### Background
 
--   [Nimble Table Issue](https://github.com/ni/nimble/issues/283)
--   [Nimble Table Research Issue](https://github.com/ni/nimble/issues/285)
--   [Nimble Table Project Page](https://github.com/orgs/ni/projects/11)
--   [Nimble Table Research Lucid Board](https://lucid.app/lucidspark/31f2314d-dd8e-46fd-8fc1-6e9f66700bb3/edit?viewport_loc=-6023%2C-26311%2C35411%2C20921%2CloaYwcZLRray4&invitationId=inv_38839ad5-72b2-4975-ab7a-6d8be33c960c)
+[Nimble issue #283](https://github.com/ni/nimble/issues/283)
 
-## Implementation / Design
+[Nimble Table Tech Stack Document](./tech-stack.md)
 
-We have decided to build our table using [TanStack Table](https://tanstack.com/table/v8) as the data model, and create the UI ourselves using typical FAST-based templates. Using TanStack Table provides a lot of lift for the state management/API of the table, including:
+[Nimble Table Research Document](https://lucid.app/lucidspark/31f2314d-dd8e-46fd-8fc1-6e9f66700bb3/edit?viewport_loc=-1060%2C-25492%2C20822%2C12325%2CloaYwcZLRray4&invitationId=inv_38839ad5-72b2-4975-ab7a-6d8be33c960c)
 
--   Row selection
--   Column sizing/sorting/ordering/pinning
+[Table visual design](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/b9cee5e2-49a4-425a-9ed4-38b23ba2e313/)
+
+### Non-goals
+
+-   The `nimble-table` will not offer spreadsheet-like features such as infinite columns, or summary functions.
+
+-   We do not intend for the `nimble-table` to handle data sets of 50K or more on the client-side. Visualizing such sizes of data sets are meant to be enabled through various "data virtualiazation" means, such as the server only sending over subsets of the data at a time, appropriately filtered down to a reasonable size.
```

Comment:
nit: in the performance section we use 10K as the upper limit so may as well be consistent.
```suggestion
-   We do not intend for the `nimble-table` to handle data sets of 10K or more on the client-side. Visualizing such sizes of data sets are meant to be enabled through various "data virtualiazation" means, such as the server only sending over subsets of the data at a time, appropriately filtered down to a reasonable size.
```